### ISSUE TITLE
Add file-level information to packages in SPDX reports

### ISSUE THREAD

**Issue by sschuberth** [oss-review-toolkit/ort#5289]
The issue also mentions the export of file level information, and that is not yet solved

_Originally posted by @maxhbr in https://github.com/oss-review-toolkit/ort/issues/4099#issuecomment-1110671671_

**Comment by sschuberth** [comment]
As file-level information (like detected licenses etc.) could easily blow up the SPDX report a lot, adding these details should probably be hidden behind a reporter-specific option.

**Comment by daniel-kr** [comment]
This works by now, doesn't it? There is the option `file.information.enabled` that enables file-level information in SPDX reports. So, probably, this issue can be closed?

**Comment by sschuberth** [comment]
> So, probably, this issue can be closed?

Indeed, thanks @daniel-kr! For reference, this was implemented in https://github.com/oss-review-toolkit/ort/pull/7255.
